IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -67,23 +71,43 @@ def compute_ims(accelerations, configuration, adv_im_out_dir):
             subprocess.run(script)
 
 
-def get_acc_filename(folder, stat, component):
-    # returns the same filename structure as timeseries.seis2txt
+def get_acc_filename(folder: str, stat: str, component: str):
+    """
+    Returns the filename for the acceleration file
+
+    Parameters
+    ----------
+    folder : str
```

Comment:
Could this be Path? I believe the rest of code is not really pathlib ready?

Reply:
I mainly didnt change the input type because this function is still used elsewhere in the code. My concern is my task of fixing the filename change to the qcore repo then expands into changing more things down the rabbithole as we go further and further

Reply:
Do we even need this function at all??? Surely in the one place it's being used we can just do `folder / f'{stat}.{component}'`

Reply:
but yeah I agree, we can just keep it how it is, if we fixed the whole repo every time we would never get anything done.